--- origsrc/Gtk2-GLExt-0.90/Makefile.PL	2004-10-28 19:18:43.000000000 -0500
+++ src/Gtk2-GLExt-0.90/Makefile.PL	2008-03-17 23:30:28.953125000 -0500
@@ -39,6 +38,8 @@
    exit 1; # not reached
 }
 
+mkdir 'build', 0777;
+
 %pkgcfg = ExtUtils::PkgConfig->find ('gtkglext-1.0 >= '
 				     .$build_reqs{GtkGLExt});
 
@@ -48,7 +49,6 @@
 	Glib::MakeHelper->do_pod_files (@xs_files),
 );
 
-mkdir 'build', 0777;
 
 #
 # autogeneration
